themes/getting-started/theme-yml.md

```diff
@@ -34,6 +34,43 @@ Users will be able to choose the layout for each page from the theme's settings
         description: Great for CMS pages to show advertisements on the side
 ```
 
+## Parent / child settings
+
+{{% notice note %}}
+Find mode informations about [Theming inheritance - Parent/child themes]({{<relref "/8/themes/reference/template-inheritance/parent-child-feature">}})
+{{% /notice %}}
+
+To set the parent theme, set the parent key to the theme's name: 
+
+```yaml
+parent: classic
+```
+
+### use_parent_assets 
+
+From {{< minver v="8.0" >}} , the setting `use_parent_assets` is functional.
```

Comment:
As we are inside the 8 part of the doc, it's needed to say that the minver is 8 ?

Reply:
Since the option was available in 1.7 but not working, i thought it was needed ?